lib/main.dart

```diff
@@ -1,16 +1,22 @@
 import 'package:flutter/material.dart';
+import 'package:isar/isar.dart';
 import 'package:kana_to_kanji/src/app.dart';
 import 'package:kana_to_kanji/src/core/repositories/settings_repository.dart';
+import 'package:kana_to_kanji/src/core/services/groups_service.dart';
+import 'package:kana_to_kanji/src/core/services/kana_service.dart';
 import 'package:kana_to_kanji/src/locator.dart';
 
 void main() async {
   setupLocator();
   WidgetsFlutterBinding.ensureInitialized();
 
+  await Isar.initialize();
   locator.allReadySync();
 
   // Load the user settings. Need to be run before the runApp for the theme mode.
   await locator<SettingsRepository>().loadSettings();
+  await locator<GroupsService>().loadCollection();
+  await locator<KanaService>().loadCollection();
```

Comment:
Is it necessary to load the collection before starting the application? 
Can't we do that on the fly? Also if it's necessary, can you change the locator to register them as lazy async singleton?

Reply:
C'est retiré, ça avait été mis là car j'avais des soucis avec isar et il mettait beaucoup de temps pour gérer ses trucs, du coup j'avais rajouté ça ici pour tenter de régler le problème.
Le problème originel ayant été réglé, j'ai pu retester sans et ça roule.